### DISCUSSION TITLE
Chamilo 1.11.26, show the users progress for each course

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5155] [General]
Hi, is there a way to show for each course the completion percentage for the current user in the courses home page? Can't find any related settings and we're struggling to find the necessary data into the template.
Do we have to extend the controller to get the data?

Thanks in advance

**Comment by jgnosolored** [comment]
Hello @a-mian-lateral

Check your app/config/configuration.php file, there are variables that you can configure
Example:
https://github.com/chamilo/chamilo-lms/blob/1.11.x/main/install/configuration.dist.php
```
//Show student progress in My courses page
//$_configuration['course_student_info']['score'] = false;
//$_configuration['course_student_info']['progress'] = false;
//$_configuration['course_student_info']['certificate'] = false;
```

I hope it helps you.

**Reply by a-mian-lateral** [reply]
Thank you, it helps a lot. So, by taking a look at that file, I suppose there are many settings that cannot be configured from the web interface. Is that correct?

**Reply by jgnosolored** [reply]
Hello. It's right. 

There are configurations that can only be modified by editing the app/config/configuration.php file. 
It is always recommended to create a backup copy before making any modifications to it.

Best regards.

**Reply by ywarnier** [reply]
Most of the settings available from configuration.php today will appear in the web interface (for admins) in Chamilo 2.0 and 2.1.
Chamilo 2.0 is not to be considered as a full replacement for Chamilo 1.11.26 (many features will *not* be present in the first release) but there are probably already a few hundred settings that will be visible in the admin panel and work correctly.
In Chamilo 1.11, configuration.php is like the secret source of magic that you only discover if you dive a bit deeper.
